spec/lib/vcr/errors_spec.rb

```diff
@@ -31,7 +31,7 @@ def request_with(values)
           )
         end
 
-        it 'mentions that there is no current cassette' do
+        it 'mentions that there is no cassette' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.